packages/back-end/src/integrations/SqlIntegration.ts

```diff
@@ -1542,12 +1569,13 @@ export default abstract class SqlIntegration
       displayNameColumn,
       includesPagesTable,
       includesScreensTable,
+      groupByColumns,
+      eventHasUniqueTable,
+      dateFormat,
     } = this.getSchemaFormatConfig(schemaFormat);
 
-    const currentDateTime = new Date();
-    const SevenDaysAgo = new Date(
-      currentDateTime.valueOf() - 7 * 60 * 60 * 24 * 1000
-    );
+    const today = formatDate(new Date(), dateFormat);
+    const sevenDaysAgo = formatDate(subDays(new Date(), 100), dateFormat);
```

Comment:
Is `100` right in `subDays(new Date(), 100)`?
```suggestion
    const sevenDaysAgo = formatDate(subDays(new Date(), 100), dateFormat);
```

Reply:
Oh gosh - no - an artifact from testing. Updated [here](https://github.com/growthbook/growthbook/pull/1423/commits/67f2f5b71a13d26c5507a8eb1448a53f13685438).